lms/djangoapps/verify_student/management/commands/idvs_of_march_resubmit.py

```diff
@@ -0,0 +1,89 @@
+"""
+Django admin commands related to verify_student
+"""
+
+
+import logging
+from django.core.management.base import BaseCommand
+
+from lms.djangoapps.verify_student.models import SoftwareSecurePhotoVerification, SSPVerificationRetryConfig
+
+log = logging.getLogger('retry_photo_verification')
+
+
+class Command(BaseCommand):
+    """
+    This method finds those PhotoVerifications with a status of
+    'submitted' from 2023-03-01 to 2023-03-28 and attempts to verify them.
+    """
+    args = "<SoftwareSecurePhotoVerification id, SoftwareSecurePhotoVerification id, ...>"
+    help = (
+        "Retries SoftwareSecurePhotoVerifications passed as "
+        "arguments, or if no arguments are supplied, all that "
+        "are in a state of 'submitted' from 2023-03-01 to 2023-03-28"
+    )
+
+    def add_arguments(self, parser):
+
+        parser.add_argument(
+            '--verification-ids',
+            dest='verification_ids',
+            action='store',
+            nargs='+',
+            type=str,
+            help='verifications id used to retry verification'
+        )
+
+        parser.add_argument(
+            '--args-from-database',
+            action='store_true',
+            help='Use arguments from the SSPVerificationRetryConfig model instead of the command line.',
+        )
+
+    def get_args_from_database(self):
```

Comment:
If you are doing a separate management command, you don't need to support args from the db anymore